aten/src/ATen/native/vulkan/glsl/div_scalar.glsl

```diff
@@ -0,0 +1,26 @@
+#version 450 core
+#define PRECISION $precision
+
+layout(std430) buffer;
+
+/* Qualifiers: layout - storage - precision - memory */
+
+layout(set = 0, binding = 0) uniform PRECISION restrict writeonly image3D   uOutput;
+layout(set = 0, binding = 1) uniform PRECISION                    sampler3D uInput;
+layout(set = 0, binding = 2) uniform PRECISION restrict           Block {
+  ivec3 size;
+  float other;
+} uBlock;
+
+layout(local_size_x_id = 0, local_size_y_id = 1, local_size_z_id = 2) in;
+
+void main() {
+  const ivec3 pos = ivec3(gl_GlobalInvocationID);
+
+  if (all(lessThan(pos, uBlock.size))) {
+    imageStore(
+        uOutput,
+        pos,
+        texelFetch(uInput, pos, 0) / uBlock.other);
+  }
+}
```

Comment:
Remove this entire codepath and re-use multiply by (1.0f / other).  Not only will that have the benefit of code reuse, it will also reduce binary size and multiplication is also typically faster than division.